### DISCUSSION TITLE
k8s restart skywalking-oap error

### DISCUSSION THREAD

**Discussion by xiangzirui** [apache/skywalking#12118] [General]
I am trying to restart skywalking-oap. I found that the initialization of the container failed. The following error is reported. Please help me solve it.
skywalking oap version:9.7.0
`2024-04-16 12:57:39,176 org.apache.skywalking.oap.server.core.storage.model.ModelInstaller 43 [main] INFO  [] - table: alarm_record does not exist. OAP is running in 'no-init' mode, waiting... retry 3s later.`
![Dingtalk_20240416210718](https://github.com/apache/skywalking/assets/49627024/146fd0fa-d153-4ffe-8763-4684c2ddfabb)
![Dingtalk_20240416210823](https://github.com/apache/skywalking/assets/49627024/de32faa6-5539-4418-ad12-c82a3f005f3e)
![Dingtalk_20240416210750](https://github.com/apache/skywalking/assets/49627024/5ebd8af6-cafa-4219-80ff-5a39ecec7f90)

**Comment by wu-sheng** [comment]
Which database are you running? Did you wipe the storage then restart OAP?

**Reply by xiangzirui** [reply]
I deployed using the default elasticsearch for helm, and I didn't wipe the storage before restart.

**Reply by wu-sheng** [reply]
The error shows you did. Maybe you don't setup persistent storage for elasticsearch somehow, or it is unhealthy.

**Reply by xiangzirui** [reply]
Well, when there was a problem at first, when I tried the recovery steps searched on the network, I mistakenly deleted the previous replica set of elasticsearch, resulting in data loss. Is there any good way I can restore skywalking now? the loss of the previous monitoring data has little impact on me.

**Reply by wu-sheng** [reply]
Follow the doc and run reinit, https://github.com/apache/skywalking-helm?tab=readme-ov-file#rerun-oap-init-job
